publication/src/main/scala/hmda/publication/reports/protocol/disclosure/D53Protocol.scala

```diff
@@ -0,0 +1,58 @@
+package hmda.publication.reports.protocol.disclosure
+
+import hmda.model.publication.reports.{ ApplicantIncome, Disposition, MSAReport }
+import hmda.model.publication.reports.ReportTypeEnum.Disclosure
+import hmda.publication.reports.disclosure.D53
+import hmda.publication.reports.protocol.{ ApplicantIncomeProtocol, MSAReportProtocol, ReportTypeEnumProtocol }
+
+import spray.json._
+import spray.json.DefaultJsonProtocol
+
+object D53Protocol
```

Comment:
This is almost an exact copy of `D51Protocol`.  Is there any way to reduce this duplication?

Reply:
Excellent question. This duplication is bothering me too. (side note: there's also blatant duplication in `DisclosureReportGenerators`, since each generator is almost an exact copy of the others).

I'd love to come up with a better system for making these, and I want to give this the thought it deserves. Should we keep this PR open while I think about that?